.pre-commit-config.yaml

```
@@ -14,6 +14,7 @@ repos:
     hooks:
     - id: flake8
       language_version: python3
+      additional_dependencies: ["flake8-pytest-style"]
```

Do you mind using a multiline sequence style? It reads better and the quotes aren't really necessary. Plus adding extra things like `flake8-docstrings` is easier in the future.
```suggestion
      additional_dependencies:
      - flake8-pytest-style
```

---

Yes, that would be better. Thanks